modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -109,6 +110,10 @@ public void test() throws SQLException {
     }
 
     private void performSimpleTest(HikariDataSource dataSource) throws SQLException {
+        if (jdbcUrl.startsWith("jdbc:tc:db2:")) {
+            return; // The "SELECT 1" statement is not valid in DB2
```

Comment:
is there a similar query in DB2? Perhaps we can use something like:
```java
String query = "SELECT 1";
if (jdbcUrl.startsWith("jdbc:tc:db2:")) {
    query = "SELECT ..."; // DB2-friendly query here
}

Reply:
yes, we could use `SELECT 1 FROM SYSIBM.SYSDUMMY1` which is what `Db2Container.getTestQueryString()` returns

Reply:
Nice! Let's use it here 👍